tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -916,6 +916,12 @@ describe("Test Application deployment based on 'clusterGroupSelector'", { tags:
       test_explanation: "multiple-apps",
       bundle_count: '2 / 2',
     },
+    {
+      qase_id: 32,
+      app: 'single-app',
+      test_explanation: "single-app to the third cluster",
```

Comment:
Suggestion:

Maybe add Fleet number at the beginning of test
Rethink second `to` (maybe as above)

![image](https://github.com/user-attachments/assets/75af5da6-47c2-493f-9f7d-0df3231a5faf)

Reply:
Updated test names and added the Fleet-test no. in the beginning.